admin/src/main/scala/io/buoyant/admin/Admin.scala

```diff
@@ -114,7 +115,7 @@ class Admin(val address: SocketAddress) {
         log.debug(s"admin: $url => ${service.getClass.getName}")
         muxer.withHandler(url, service)
     }
-    notFoundView.andThen(muxer)
+    HeadFilter andThen notFoundView andThen muxer
```

Comment:
we tend to prefer dots and parens over spaces... but this is probably fine in this contextx